src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public void enterAllSongs() {
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
+        WebElement allSongs = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a[href='#!/songs']")));
+        allSongs.click();
+
+        // WebElement allSongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
+        //allSongs.click();
+
+        //Thread.sleep(1000);
+        // allSongs.click();
+        //
```

Comment:
**I recommend not using code comments**: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.